models/database.go

```diff
@@ -692,8 +692,17 @@ var databaseSchema = [][]string{
 			WHERE 'mongo_db_tls_options' is not null AND jsonb_typeof(mongo_db_tls_options->'stats_collections') = 'string'`,
 	},
 	59: {
+		`DELETE FROM percona_sso_details WHERE organization_id = ''`,
```

Comment:
this migration makes no sense because empty organization_id won't break migration, but null in value will, so we should check for null value as well.

Reply:
changed to `IS NULL`